src/EA.Weee.Web/Areas/Scheme/Mappings/ToViewModels/ViewTransferNoteViewModelMap.cs

```diff
@@ -59,6 +59,8 @@ public ViewTransferNoteViewModel Map(ViewTransferNoteViewModelMapTransfer source
 
             var model = new ViewTransferNoteViewModel
             {
+                ApprovedRecipientDetails = source.TransferEvidenceNoteData.ApprovedTransfererDetails,
```

Comment:
@GuidoWorkDev Dont do it this way.... Set it the same way as was done for the evidence note then we dont have to change the view,